src/jit/alloc.cpp

```diff
@@ -81,11 +83,8 @@ size_t ArenaAllocator::getDefaultPageSize()
 // ArenaAllocator::ArenaAllocator:
 //    Default-constructs an arena allocator.
 ArenaAllocator::ArenaAllocator()
-    : m_memoryManager(nullptr)
-    , m_firstPage(nullptr)
-    , m_lastPage(nullptr)
-    , m_nextFreeByte(nullptr)
-    , m_lastFreeByte(nullptr)
+    : m_memoryManager(nullptr), m_firstPage(nullptr), m_lastPage(nullptr), m_nextFreeByte(nullptr),
```

Comment:
But can't we at least tell it not to reformat? VC++'s formatter has no trouble with these, it only fixes indentation if it's messed up.